zebrad/src/components/mempool/storage.rs

```diff
@@ -103,4 +103,12 @@ impl Storage {
             .filter(|tx| tx_ids.contains(&tx.id))
             .collect()
     }
+
+    /// Returns the set of [`UnminedTxId`]s matching ids in the rejected list.
+    pub fn rejected_transactions(self, tx_ids: HashSet<UnminedTxId>) -> Vec<UnminedTxId> {
```

Comment:
I think it will be good if we can include a very basic test for this as part of https://github.com/ZcashFoundation/zebra/blob/main/zebrad/src/components/mempool/storage/tests.rs#L19 or in a new one.

Reply:
https://github.com/ZcashFoundation/zebra/pull/2718/commits/ffb7b1229b489d916728cca2dab3b34f56d6c617